packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap

```diff
@@ -1339,6 +1340,7 @@ exports[`StatsTestCases should print correct stats for limit-chunk-count-plugin
       "outputPath": "<PROJECT_ROOT>/tests/statsCases/limit-chunk-count-plugin/dist",
       "publicPath": "",
       "rspackVersion": "0.3.8",
+      "time": 64,
```

Comment:
Remove this

Reply:
I will check out the default behavior of webpack